src/python/gem5/resources/client.py

```diff
@@ -151,6 +151,7 @@ def get_resource_json_obj(
     resource_version: Optional[str] = None,
     clients: Optional[List[str]] = None,
     gem5_version: Optional[str] = core.gem5Version,
+    proxy_context: Optional[object] = None,
```

Comment:
I don't think the type is correct. I believe it should be https://docs.python.org/3/library/ssl.html#ssl.SSLContext